pyvera/__init__.py

```diff
@@ -284,10 +284,25 @@ def __init__(self, json_obj, vera_controller):
             else:
                 self.name = 'Vera Device ' + str(self.device_id)
 
-    def get_payload_parameter_name(self, name):
+    @staticmethod
```

Comment:
This is overridden by some derived classes, some of which then call the base class version, so I don't think it's right to make this `@staticmethod`, although your python may be better than mine...